app/domain/authentication/authn_jwt/jwt_configuration_dummy_vendor.rb

```diff
@@ -2,16 +2,16 @@ module Authentication
   module AuthnJwt
     # Mock JWTConfiguration class to use it to develop other part in the jwt authenticator
     class JWTConfigurationDummyVendor < JWTConfigurationInterface
-      def get_identity
-        return "cucumber"
+      def get_conjur_id
+        "cucumber"
       end
 
       def validate_restrictions
-        return true
+        true
       end
 
-      def validate_and_decode
-        return true
+      def validate_and_decode_token
```

Comment:
the `and` amplifies that this method hides double meaning.
Maybe split it into `validate_token` and `decode_token`?

Reply:
i am guessing that this method will use the third-party that we use today that validates and decodes the token in one shot.